tests/collect-cov.sh

```diff
@@ -23,8 +23,10 @@
 
 set -e
 
-lcov --capture --directory `pwd` --compat-libtool --output-file coverage.info --rc geninfo_auto_base=1
+lcov --capture --directory `pwd` --compat-libtool --output-file coverage.info
 lcov --extract coverage.info "${top_srcdir}/*" --output-file coverage.info
 lcov --remove coverage.info "${top_srcdir}/modules/afmongodb/mongo-c-driver/*" --output-file coverage.info
```

Comment:
Not related to this PR, but the `mongo-c-driver` and `rabbitmq-c` submodules are no longer part of the projects so there is no need to exclude them.

Reply:
@Kokan: indeed, we should remove them.

Reply:
@Kokan : #2525